### PR TITLE
fix: make semantic changes to commands

### PR DESCRIPTION
**Pull request by Thunderbottom**
* add missing `__main__` call to commands.py
* remove unnecessary imports
* fix backup WITH_FILES logic
* follow python semantics (?)

### CODE REVIEW

**Comment by revant** [comment]
new command has error

```
sh: 1: _fcadc07dbdbc21db: not found
ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '* TO '_fcadc07dbdbc21db'@'%'' at line 1
```

```sh
❯ docker run \
    -e "SITE_NAME=test.localhost" \
    -v frappebench00_sites-vol:/home/frappe/frappe-bench/sites \
    --network frappebench00_default \
    frappe/frappe-worker:edge new
Created user _fcadc07dbdbc21db
Created database _fcadc07dbdbc21db
Granted privileges to user _fcadc07dbdbc21db and database _fcadc07dbdbc21db
Starting database import...
Imported from database /home/frappe/frappe-bench/apps/frappe/frappe/database/mariadb/framework_mariadb.sql

Installing frappe...
Updating DocTypes for frappe        : [========================================]
Updating country info               : [========================================]sh: 1: _fcadc07dbdbc21db: not found
ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '* TO '_fcadc07dbdbc21db'@'%'' at line 1

*** Scheduler is disabled ***

```

**Comment by revant** [comment]
If you have native frappe-bench installed you can debug locally

1. sym-link commands

```sh
cd path/to/frappe-bench
ln -s path/to/frappe_docker/build/common/commands ./
```

2. `cd sites`
3. `. ../env/bin/activate`
4. `python ../commands/new.py`

**Comment by Thunderbottom** [comment]
should be fixed now!

File: build/common/commands/check_connection.py [hunk 1]
```diff
@@ -8,30 +10,33 @@
...
     finally:
         s.close()
 
+
 def check_host(ip, port, retry=10, delay=3, print_attempt=True):
     ipup = False
     for i in range(retry):
         if print_attempt:
-            print("Attempt {i} to connect to {ip}:{port}".format(ip=ip,port=port,i=i+1))
+            print("Attempt {i} to connect to {ip}:{port}".format(ip=ip, port=port, i=i+1))
```
[the remark this answers is not in the document]
Comment (revant) [reply build/common/commands/check_connection.py:31]: Yes. f-strings can be used. Worker image is based on `bitnami/python:latest-prod` which is 3.7

**Pull request merged**